facebook-commerce.php

```diff
@@ -3753,129 +3546,6 @@ function ajax_display_test_result() {
 		wp_die();
 	}
 
-
-	/**
-	 * Schedules a recurring event to sync products.
-	 *
-	 * @deprecated 1.10.0
-	 */
-	function ajax_schedule_force_resync() {
-
-		wc_deprecated_function( __METHOD__, '1.10.0' );
-		die;
-	}
```

Comment:
The function stills being called in line 335
https://github.com/woocommerce/facebook-for-woocommerce/blob/84d2e84eaa4e9667c01b2b37748386d9a4d17cd2/facebook-commerce.php#L334-L336

Reply:
Deleted.